backend/grant/proposal/views.py

```diff
@@ -358,8 +348,8 @@ def get_proposal_update(proposal_id, update_id):
 @limiter.limit("5/day;1/minute")
 @requires_team_member_auth
 @body({
-    "title": fields.Str(required=True),
-    "content": fields.Str(required=True)
+    "title": fields.Str(required=True, validate=lambda p: 3 <= len(p) <= 30),
+    "content": fields.Str(required=True, validate=lambda p: 5 <= len(p) <= 3000),
```

Comment:
3,000 isn't all that long, are you sure we want to limit it to that? We should also probably add frontend validation as well.